web/index.php

```diff
@@ -67,10 +67,15 @@
 // Use relative URL's instead
 define( "ZM_BASE_URL", "" );
 
+$system_timezone = preg_replace( "/\r|\n/", '', file_get_contents('/etc/timezone') );
```

Comment:
The file /etc/timezone is Debian-specific. It does not exist on an rpm based distro